src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -175,7 +175,7 @@ public ChimericAlignment(final AlignmentInterval intervalWithLowerCoordOnContig,
             final ChimericAlignment chimericAlignment = new ChimericAlignment(current, next, insertionMappings,
                     alignedContig.contigName, referenceDictionary);
             // the following check/filter is due to the fact that simple translocations are to be handled in a different code path
-            if (chimericAlignment.isNotSimpleTranslocation())
+            if (chimericAlignment.isNeitherSimpleTranslocationNorIncompletePicture())
```

Comment:
using correct filtering predicate now, not being simple translocation could mean not having a complete picture (see logic below). This is the reason the 1st line of defense is not working.